code/modules/events/black_hole.dm

```diff
@@ -142,7 +143,7 @@
 			return
 
 		for (var/atom/X in range(7,src))
-			if (X == src || (X.event_handler_flags & IMMUNE_SINGULARITY))
+			if (X.event_handler_flags & IMMUNE_SINGULARITY)
```

Comment:
The two changes in this file mean these black hole things will not pull each other towards each other, I believe? Verifying if intended behavior, as not part of the PR description. Doesn't seem like a big deal, just noting.

Reply:
It's part of the PR description.